printrun/gui/__init__.py

```diff
@@ -202,8 +204,7 @@ def createTabbedGui(self):
         self.Bind(wx.EVT_CLOSE, self.kill)
 
         # Custom buttons
-        if wx.VERSION > (2, 9): self.cbuttonssizer = wx.WrapSizer(wx.HORIZONTAL)
-        else: self.cbuttonssizer = wx.GridBagSizer()
+        self.cbuttonssizer = wx.WrapSizer(wx.HORIZONTAL)
         self.cbuttonssizer = wx.GridBagSizer()
```

Comment:
Is it just me or does this line invalidate the effect of the previous line?

Reply:
It does, however, the behavior stays the same :)

Should I remove the previous one?